packages/next/src/templated-launcher.ts

```diff
@@ -9,9 +9,9 @@ if (!process.env.NODE_ENV) {
   process.env.NODE_ENV = region === 'dev1' ? 'development' : 'production';
 }
 
+// NOTE: `eval('require')` is necessary to avoid bad transpilation to `__webpack_require__`
```

Comment:
Are we sure this one is necessary? I don't believe this file is compiled by webpack.

Reply:
@ijjk could maybe confirm.

Reply:
Yeah this one shouldn't be touched by `ncc` just `tsc` x-ref: https://github.com/vercel/vercel/blob/0b56caba45c4ab0a74e7ae0a451cda033db2db04/packages/next/build.js#L9